### DISCUSSION TITLE
No autostart of hyperion after update to alpha 11?

### DISCUSSION THREAD

**Discussion by N1co93** [hyperion-project/hyperion.ng#1349] [Releases] [answered]
Hello, 

I just updated from Alpha 10 to Alpha 11, with this file: `Hyperion-2.0.0-alpha.11-Linux-armv7l.deb`
After this update I noticed, that autostart of hyperion doesn't work anymore.

I also noticed, that the following command doesn't work anymore `sudo systemctl status hyperiond@pi.service`

Do I have to do some manual changes after the last update?

Thank you in advance

**Comment by Paulchen-Panther** [comment, marked as answer]
> I also noticed, that the following command doesn't work anymore `sudo systemctl status hyperiond@pi.service`
> 
> Do I have to do some manual changes after the last update?

Yes, you're right about that. As of Alpha 11, the Hyperion service is only applied automatically for non-gui users, since a program entry is made in the Start menu when the user interface is available and Hyperion can thus be started manually.
Since we are totally understaffed in the Hyperion team, and still have to do our real work to earn money, this hint has not yet been incorporated into the documentation.
The Hyperion service is also only called hyperion instead of hyperion(d)(eamon).

Greetings Paulchen

**Reply by N1co93** [reply]
Thank you for your reply and also for the hard work.
I really appreciate it. Keep up the good work! 

I just send you a donation, I hope you and the team can enjoy some beer with it 🍻
